data/human/human missions.txt

```diff
@@ -5856,8 +5856,8 @@ mission "Quicksilver Mixup 2"
 		has "event: quicksilver mixup 2 available"
 	on offer
 		conversation
-			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
-			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
+			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet walks up to you after ending a hurried conversation with another pilot.`
```

Comment:
```suggestion
			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet walks up to you after ending a hurried conversation with another pilot.`
```
Not necessary; the original sentence is correct. This is more a writing style choice than a fix.